src/components/settings/AddDeleteSetting.js

```diff
@@ -44,9 +44,8 @@ class AddDeleteSetting extends Component {
     })
   }
 
-  onDeleteItem(event) {
+  onDeleteItem(idx) {
```

Comment:
What is wrong with using dataset properties? https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes#JavaScript_Access